plugins/sql.c

```diff
@@ -668,6 +670,7 @@ static char *sql_create_statement(const char *statement, const char *prop,
     rlen = (int)strlen(realm);
     plen = (int)strlen(prop);
     vlen = (int)sql_len(value);
+    alen = (int)sql_len(remoteaddrport);
     
```

Comment:
you also need to add at line 681 `biggest = sql_max(biggest, alen);` otherwise you can cause a buffer overflow.